scripts/generate_metric_enums.py

```diff
@@ -26,6 +26,8 @@
     "LATENCY",
     "ROWS_RETURNED",
     "OPERATOR_NAME",
+    "SYSTEM_PEAK_BUFFER_MANAGER_MEMORY",
+    "SYSTEM_PEAK_TEMP_DIRECTORY_SIZE",
```

Comment:
Nit to be slightly less verbose: Maybe just call it `SYSTEM_PEAK_BUFFER_MEMORY` and `SYSTEM_PEAK_TEMP_DIR_SIZE`?